mod/missions_collab/views/default/forms/missions/advanced-search-form.php

```diff
@@ -64,7 +64,9 @@
 
 $number_of_rows = elgg_get_plugin_setting('advanced_element_limit', 'missions');
 $content = '<div class="form-group">';
+//$content .= '<label for="mission-emphasis-extra col-sm-3" class="required text-right" aria-required="true">'
```

Comment:
I'm guessing this is just from an experiment that needs to be cleaned up

Reply:
Yes, has been removed